pkgs/development/python-modules/msgpack/default.nix

```diff
@@ -0,0 +1,31 @@
+{ buildPythonPackage
+, fetchPypi
+, pytest
+, lib
+}:
+
+buildPythonPackage rec {
+    pname = "msgpack";
+    version = "0.5.4";
+    name = "${pname}-${version}";
+
+    src = fetchPypi {
+      inherit pname version;
+      sha256 = "13ckbs2qc4dww7fddnm9cw116j4spgxqab49ijmj6jr178ypwl80";
+    };
+
+    propagatedBuildInputs = [ ];
+
+    checkPhase = ''
+      py.test
+    '';
+
+    checkInputs = [ pytest ];
+
+    meta = {
+      homepage = https://github.com/msgpack/msgpack-python;
+      description = "MessagePack serializer implementation for Python";
+      license = lib.licenses.asl20;
+      # maintainer =  ?? ;
```

Comment:
How about adding yourself as maintainer? Btw, the attribute should be called `maintainers`.

Reply:
Hmm, I don't use it myself. Just wanted to update the version to get the neovim python client going again at version 0.2.1. Isn't that a bit weak motivation to declare myself maintainer and actually I am just trying nixos at the moment. I don't know how long I will be around at nixos.

Reply:
Well, if you're not sure, then better remove the line completely. But feel free to add yourself once you're convinced to stick with NixOS :)
At the very least, you're using it because you're using neovim. So a maintained package does benefit you (and others).